homeassistant/components/automation/__init__.py

```diff
@@ -38,13 +38,15 @@
 CONF_ACTION = 'action'
 CONF_TRIGGER = 'trigger'
 CONF_CONDITION_TYPE = 'condition_type'
+CONF_INITIAL = 'initial'
```

Comment:
Oh.

Let's still stick with `initial_state`, maybe one day we'll convert the others (but only if we have a need to change their config).